clang/include/clang/Tooling/Inclusions/StandardLibrary.h

```diff
@@ -29,6 +29,27 @@ class NamespaceDecl;
 class DeclContext;
 namespace tooling {
 namespace stdlib {
+enum Version {
```

Comment:
We already have enums for standards versions elsewhere, I don't want us maintaining ANOTHER one.  So I'm against this enum entirely.